packages/frontend-core/src/components/grid/cells/HeaderCell.svelte

```diff
@@ -41,6 +48,17 @@
   $: descendingLabel = ["number", "bigint"].includes(column.schema?.type)
     ? "high-low"
     : "Z-A"
+  $: searchable = isColumnSearchable(column)
+  $: searching = searchValue != null
+  $: debouncedUpdateFilter(searchValue)
+
+  const isColumnSearchable = col => {
+    const type = col.schema.type
+    return (
+      searchableTypes.includes(type) ||
+      (type === "formula" && col.schema.formulaType === "static")
```

Comment:
Same here, if we use the fields from `types` we will have safer refactors code as we will not depend on magic strings